src/components/progress-report/workflow/handlers/progress-report-workflow-notification.handler.ts

```diff
@@ -76,7 +76,7 @@ export class ProgressReportWorkflowNotificationHandler
         this.prepareNotificationObject(
           reportId,
           previousStatus,
-          userId ? { userId } : { email },
+          userId ? { userId } : email ? { email } : { email: '' },
```

Comment:
Empty strings to hide type errors are a big ick for me.
Instead, in `getProjectNotifyees`, member's without email addresses can be `flatMap`'ped out.

Reply:
@CarsonF , I'm having trouble getting rid of the null email values. I pushed up a version using flatMap in the `getProjectNotifyees` method, so you can see the code.

Reply:
I couldn't get the flatMap to work with types either, not exactly sure why.
In the end, it's better to just to align the DB queries closer to our need.
See my other comments in the repo.